code/LINDA/LINDA_fire.dm

```diff
@@ -152,18 +152,18 @@
 // Garbage collect itself by nulling reference to it
 
 /obj/effect/hotspot/proc/Kill()
-	air_master.hotspots -= src
-	DestroyTurf()
-	qdel(src)
+	PlaceInPool(src)
```

Comment:
Eh, yes, it's fine, and again I didn't read through it and failed to realize that the pool will Destroy() things. My point now is primarily that everything below this shouldn't directly call an arbitrary variable when qdel will achieve the same thing and be standardized